main/webapp/modules/core/scripts/util/i18n.js

```diff
@@ -0,0 +1,39 @@
+I18NUtil = {};
+
+var lang = (navigator.language).split("-")[0];
+var dictionary = "";
+
+/*
+   Initialize i18n and load message translation file from the server.
+
+   Note that the language is set by the 'userLang' user preference setting.  You can chnage that by
+   clicking on 'Language Settings' on the landing page.
+*/
+I18NUtil.init = function (module) {
+    $.ajax({
+        url: "command/core/load-language?",
+        type: "POST",
+        async: false,
+        data: {
+            module: module,
+            lang: lang
```

Comment:
This `lang: lang` binding seems to have been commented out in most of the blocks that this new function is replacing. Are you clear on what we are changing by including this parameter?

Reply:
I looked at the implementation of core/command/load-language in LoadLanguageCommand.java, and it looks like if 'lang' is not passed it defaults to the user preference 'userLang'. If we always pass in 'lang', then the user preference 'userLang' is never used.  If people were using the 'userLang' preference to test different locales then this PR would break that.

I will remove passing the 'lang' parameter since most of the i18n init code was not passing 'lang.' I'm not sure how the 'userLang' preference was being set, and how it is working for locales other than English.

It makes sense to use the browser's locale, but if people are installing OpenRefine on their own personal desktops it might not be an issue.

Reply:
Yep, I saw that also and was changing it so that the lang passed in was a suggestion and the stored preference overrides.  Otherwise, why set a language preference in OR if the browser overrides.

I was waiting for your finished code before sending mine in.

Reply:
Makes sense, let's merge this and then @AtesComp can have a go at tweaking the logic :)